common/src/Model/Brush.h

```diff
@@ -196,14 +193,19 @@ namespace TrenchBroom {
             struct CanMoveVerticesResult {
             public:
                 bool success;
-                BrushGeometry geometry;
-
+                BrushGeometry* geometry;
```

Comment:
Yeah... I considered it, and since you are on the fence here, too, I'll switch to it. I didn't do it initially because I am now hesitant to include std headers in our header files. But I guess <memory> is an exception because it is really useful and will end up being included all over the place anyway.